clusters/app.ci/assets/bugzilla-backporter.yaml

```diff
@@ -0,0 +1,111 @@
+kind: List
+apiVersion: v1
+items:
+- kind: ImageStream
+  apiVersion: image.openshift.io/v1
+  metadata:
+    name: bugzilla-backporter
+    namespace: ci
+  spec:
+    lookupPolicy:
+      local: true
+    tags:
+    - name: latest
+      importPolicy:
+        scheduled: true
+      from:
+        kind: DockerImage
+        name: registry.svc.ci.openshift.org/ci/bugzilla-backporter
```

Comment:
Should we only merge the deployment once we actually have the image?

Reply:
Yeah! Thats the intention, but will probably have to fix stuff before thats ready.

Reply:
I think peters point is that the two things need to be seperate PRs, first the build config, second the yamls in here

Reply:
Oh ok. Got it!